Jetcaster/app/src/main/java/com/example/jetcaster/ui/player/PlayerScreen.kt

```diff
@@ -298,6 +394,42 @@ private fun PodcastDescription(
     }
 }
 
+@Composable
+private fun PodcastInformation(
+    title: String,
+    name: String,
+    summary: String,
+    nameTextStyle: TextStyle = MaterialTheme.typography.h3,
+    titleTextStyle: TextStyle = MaterialTheme.typography.h5,
```

Comment:
Nit: order of parameters. if `title` appears above `name`, I'd have `titleTextStyle` before `nameTextStyle` as well

Reply:
Ack